conversion/conversion.go

```diff
@@ -125,55 +125,67 @@ func DataConv(driver string, ioHelper *IOStreams, client *sp.Client, conv *inter
 	}
 }
 
-func driverConfig(driver string) (string, error) {
+func generateSQLConnectionStr(driver string) (string, error) {
 	switch driver {
 	case POSTGRES:
-		return pgDriverConfig()
+		return generatePGSQLConnectionStr()
```

Comment:
will this print the error to specify env variables if the source connection profile structure is used instead ?

Reply:
This part of code is only hit if ran in the legacy mode. So if source profile is specified, this part of code will never be hit. 
That said, it is a bit confusing right now because both subcommand and legacy mode code paths intersect at SchemaConv() and DataConv()  calls and supporting both modes has caused the function calls to be a little haphazard. Once we deprecate the old mode, this half of the code should be unnecessary and can be scrapped.